src/ashell/jerry-code.c

```diff
@@ -414,17 +139,14 @@ int javascript_parse_code(const char *file_name)
     buf = read_file_alloc(file_name, &size);
 
     if (buf && size > 0) {
-        // Find and load all required js modules
-        if (load_require_modules(buf)) {
-            /* Setup Global scope code */
-            parsed_code = jerry_parse((const jerry_char_t *)buf, size, false);
-            if (jerry_value_has_error_flag(parsed_code)) {
-                DBG_PRINT("Error parsing JS\n");
-                javascript_print_error(parsed_code);
-                jerry_release_value(parsed_code);
-            } else {
-                ret = 0;
-            }
+        /* Setup Global scope code */
+        parsed_code = jerry_parse((const jerry_char_t *)buf, size, false);
+        if (jerry_value_has_error_flag(parsed_code)) {
+            DBG_PRINT("Error parsing JS\n");
+            zjs_print_error_message(parsed_code, ZJS_UNDEFINED);
```

Comment:
Or if you're able to use this function directly, maybe you can replace the calls to `javascript_print_error` with this? I haven't looked in detail.

Reply:
These javascript_print functions are old and I don't think we need them anymore.  I'll remove them and use the zjs_print_error_message for our needs.